flink-table/flink-table-common/src/test/java/org/apache/flink/table/types/DataTypeTest.java

```diff
@@ -220,4 +220,21 @@ public void getFields() {
         assertEquals(Collections.emptyList(), DataType.getFields(ARRAY(INT())));
         assertEquals(Collections.emptyList(), DataType.getFields(INT()));
     }
+
+    @Test
+    public void excludeFields() {
```

Comment:
nit: usually our test methods start with `test`. please also update the methods above.